tests_integration/test_sql.py

```diff
@@ -156,3 +158,21 @@ def test_sql_execute_query_prints_query(runner):
     assert result.exit_code == 0, result.output
     assert "select 1 as A" in result.output
     assert "select 2 as B" in result.output
+
+
+@pytest.mark.integration_experimental
+def test_sql_large_lobs_in_memory_tables(runner):
+    table_name = ObjectNameProvider(
+        "table_with_default"
+    ).create_and_get_next_object_name()
+    result = runner.invoke_with_connection(
+        [
+            "sql",
+            "-q",
+            f"create or replace table {table_name}(x int, v text default x::varchar);"
+            f"select get_ddl('table', '{table_name}');"
+            f"drop table {table_name};",
+        ]
+    )
+
+    assert "VARCHAR(134217728)" in result.output
```

Comment:
Is it enough to test table creation? Should we also test that we can ingest / print such a large payload?

Reply:
TBH this should be enough.
We could attach proof of work for upload/download LOBs.
Adding such upload/download operation to integration seems overhead.

Reply:
Of course we can print and ingest large payloads. Although, this PR only test the phase 1 of this [document](https://docs.google.com/document/d/19kwrKat1_v3_hr1JsVFNtWpWDAJGhqq-hbDJr-bUtl8/edit#heading=h.jgh43aqbetoj) which rollout began this month. 

Ingesting large payload:
<img width="1638" alt="image" src="https://github.com/user-attachments/assets/859ff011-3030-445c-8033-7dee47a318dc">


Selecting a string more than 16MBs:
![image](https://github.com/user-attachments/assets/e207d2b2-d1df-44f0-b86d-b5e075f0d3ae)